packages/components/src/ListingSummary.tsx

```diff
@@ -0,0 +1,160 @@
+`;
+
+const NewsroomIcon = styled.figure`
+  background: ${colors.accent.CIVIL_GRAY_4};
+  margin: 0 17px 0 0;
+  height: 80px;
+  min-width: 80px;
+`;
+
+const MetaItem = styled.div`
+  margin: 0 0 16px;
+`;
+const MetaLabel = styled.div`
+  color: ${colors.primary.CIVIL_GRAY_1};
+  font: normal 16px/16px ${fonts.SANS_SERIF};
+  margin: 0 0 6px;
+`;
+const MetaValue = styled.abbr`
+  color: ${colors.primary.CIVIL_GRAY_1};
+  display: block;
+  font: normal 14px/17px ${fonts.SANS_SERIF};
+  max-width: 65%;
+  overflow-y: hidden;
+  text-decoration: none;
+  text-overflow: ellipsis;
+`;
+
+export interface ListingSummaryComponentProps {
+  address?: EthAddress;
+  name?: string;
+  owners?: EthAddress[];
+  description?: string;
+  listingDetailURL?: string;
+  isInApplication?: boolean | undefined;
+  canBeChallenged?: boolean | undefined;
+  inChallengePhase?: boolean | undefined;
+  inRevealPhase?: boolean | undefined;
+  appExpiry?: number | undefined;
+  commitEndDate?: number | undefined;
+  revealEndDate?: number | undefined;
+}
+
+export class ListingSummaryComponent extends React.Component<ListingSummaryComponentProps> {
+  public render(): JSX.Element {
+    return (
+      <StyledListingSummaryContainer>
+        <StyledListingSummaryHed>
+          <NewsroomIcon />
+          <div>
+            <StyledListingSummaryNewsroomName>{this.props.name}</StyledListingSummaryNewsroomName>
+            <MetaItem>
+              <MetaLabel>Owner</MetaLabel>
+              <MetaValue title={this.props.owners![0]}>{this.props.owners![0]}</MetaValue>
+            </MetaItem>
+
+            {this.renderPhaseLabel()}
+
+            {this.renderPhaseCountdown()}
+
+            <InvertedButton size={buttonSizes.SMALL} to={this.props.listingDetailURL}>
+              View Details
+            </InvertedButton>
+          </div>
+        </StyledListingSummaryHed>
+        <StyledListingSummaryDek>{this.props.description}</StyledListingSummaryDek>
+      </StyledListingSummaryContainer>
+    );
+  }
+
+  private renderPhaseLabel = (): JSX.Element | undefined => {
+    if (this.props.isInApplication) {
+      return <AwaitingApprovalStatusLabel />;
+    } else if (this.props.inChallengePhase) {
+      return <CommitVoteStatusLabel />;
+    } else if (this.props.inRevealPhase) {
+      return <RevealVoteStatusLabel />;
+    }
+    return;
+  };
+
+  private renderPhaseCountdown = (): JSX.Element | undefined => {
+    let expiry: number | undefined;
+    if (this.props.isInApplication) {
+      expiry = this.props.appExpiry;
+    } else if (this.props.inChallengePhase) {
+      expiry = this.props.commitEndDate;
+    } else if (this.props.inRevealPhase) {
+      expiry = this.props.revealEndDate;
+    }
+
+    const warn = this.props.inChallengePhase || this.props.inRevealPhase;
+
+    if (expiry) {
+      return <CountdownTimer endTime={expiry!} warn={warn} />;
+    }
+
+    return;
+  };
+}
+
+export interface ListingSummaryListComponentProps {
```

Comment:
even though it's really small, I'd probably prefer if the list component is separated into its own file. also the `any[]` for the prop type is pretty undescriptive..